Test Card List CSV.csv

```diff
@@ -11,4 +15,4 @@ SecretRare,SecretRare_Rainbowwww_TheGayAgenda,TheGayAgenda,,,,,,,,,,,,,,,,"this,
 UltraRare,UltraRare_GuildHalleyo_this_is_a_test_of_space,this is a test of space,,,,,,,,,,,,,,,,"""item"" # 3",oh yeah,:O,5,GuildHalleyo,e
 UltraRare,UltraRare_OMbretest_Whatiscolour,Whatiscolour,Noble,675,UwU,UwU,def,2,1,dolor,lorem text,Game,yep,2,45,xd,rawr,,,UwU,6,OMbretest,f
 Uncommon,Uncommon_AltareTEST_Slimies,Slimies,,,,,,,,,,,,,,,,is this the last one?,let me check,:3,7,AltareTEST,g
-Uncommon,Uncommon_MagniTEST_Doopdedoo,Doopdedoo,Initiate,7,Zatsu,Music,ghi,1,2,sit,here,,,,,,,,,>:3,8,MagniTEST,h
\ No newline at end of file
+Uncommon,Uncommon_MagniTEST_Doopdedoo,Doopdedoo,Initiate,7,Zatsu,Music,ghi,1,2,sit,here,,,,,,,,,>:3,8,MagniTEST,h
```

Comment:
There's a stray newline here at the end, it's making a new "card" entry and that's why the console is throwing errors.